src/style/__tests__/colors.spec.js

```diff
@@ -112,38 +116,106 @@ describe('style/Colors', () => {
   });
 
   describe('generateColorPalette', () => {
+    const baseColor = '#3F88C5';
+    const tints = ['#193852', '#255379', '#316EA1', '#3F88C5', '#66A0D1', '#8DB9DD', '#B5D1E9', '#DCE9F4'];
+    const baseColorLight = '#DCE9F4';
+    const tintsLight = ['#1A3851', '#265278', '#326D9F', '#4187C3', '#68A0CF', '#8EB8DC', '#B5D1E8', '#DCE9F4'];
+    const tintsAdjustedLightness = ['#193852', '#255379', '#316EA1', '#3F88C5', '#66A0D1', '#8DB9DD', '#B5D1E9', '#DCE9F4'];
+    const tintsAdjustedSaturation = ['#1A3851', '#265278', '#326D9F', '#4187C3', '#68A0CF', '#8EB8DC', '#B5D1E8', '#DCE9F4'];
+    const saturationLevels = [-10, -10, -20, -20, -25, -25, -25, -25, -20, -10];
+    const tintsSaturationLevels = ['#1E384D', '#2D5271', '#466C8C', '#3F88C5', '#7F9EB8', '#A0B7CB', '#C1D0DD', '#E2E9EE'];
+    const tintsSaturationLevelsDarkest = ['#162837', '#223F58', '#385770', '#486E90', '#3F88C5', '#7C9CB6', '#9AB2C6', '#B7C9D7', '#D3DFE9', '#F0F5F9'];
+    const tintsAddDarkestTints = ['#12283B', '#1C405E', '#275881', '#3270A5', '#3F88C5', '#629ED0', '#86B4DA', '#A9CAE5', '#CCDFF0', '#EFF5FA'];
+
     it('should memoize calls for generateColorPalette', () => {
-      uut.getColorTint('#3F88C5', 20);
-      uut.getColorTint('#3F88C5', 50);
-      uut.getColorTint('#3F88C5', 70);
-      const cachedPalette = uut.generateColorPalette.cache.get('#3F88C5');
+      uut.getColorTint(baseColor, 20);
+      uut.getColorTint(baseColor, 50);
+      uut.getColorTint(baseColor, 70);
+      const cachedPalette = uut.generateColorPalette.cache.get(baseColor);
       expect(cachedPalette).toBeDefined();
-      expect(cachedPalette.length).toBe(8);
-      expect(cachedPalette.includes('#3F88C5')).toBe(true);
+      expect(cachedPalette.includes(baseColor)).toBe(true);
     });
 
-    it('should generateColorPalette', () => {
-      const palette = uut.generateColorPalette('#3F88C5');
-      expect(palette).toEqual(['#193852', '#255379', '#316EA1', '#3F88C5', '#66A0D1', '#8DB9DD', '#B5D1E9', '#DCE9F4']);
+    it('should generateColorPalette return 8 tints with 10 lightness increment', () => {
+      const palette = uut.generateColorPalette(baseColor);
+      expect(palette.length).toBe(8);
+      expect(palette).toContain(baseColor);
+      expect(palette).toEqual(tints);
     });
 
-    it('should generateColorPalette with adjusted saturation', () => {
-      const palette = uut.generateColorPalette('#FFE5FF');
-      expect(palette).toEqual(['#661A66', '#8F248F', '#B82EB7', '#D148D1', '#DB71DB', '#E699E6', '#F0C2F0', '#FFE5FF']);
+    it('should generateColorPalette with adjustLightness option true (default)', () => {
+      const palette = uut.generateColorPalette(baseColor, {adjustLightness: true});
```

Comment:
This is a little odd IMO, you send the default value and test it, shouldn't the test above be enough for it?

Reply:
Right, I wanted to explicitlly test it but it is redundent